usage/bigquery.go

```diff
@@ -36,14 +36,32 @@ type UsageSummaryRow struct {
 	StorageUsageMins  float64 `bigquery:"storage_usage_mins"`
 }
 
+type TotalUsageSummaryRow struct {
+	DateTs                time.Time `bigquery:"date_ts" json:"dateTs"`
+	DateS                 int64     `bigquery:"date_s" json:"dateS"`
+	WeekTs                time.Time `bigquery:"week_ts" json:"weekTs"`
+	WeekS                 int64     `bigquery:"week_s" json:"weekS"`
+	VolumeEth             float64   `bigquery:"volume_eth" json:"volumeEth"`
+	VolumeUsd             float64   `bigquery:"volume_usd" json:"volumeUsd"`
+	FeeDerivedMinutes     float64   `bigquery:"fee_derived_minutes" json:"feeDerivedMinutes"`
+	ParticipationRate     float64   `bigquery:"participation_rate" json:"participationRate"`
+	Inflation             float64   `bigquery:"inflation" json:"inflation"`
+	ActiveTranscoderCount int64     `bigquery:"active_transcoder_count" json:"activeTranscoderCount"`
+	DelegatorsCount       int64     `bigquery:"delegators_count" json:"delegatorsCount"`
+	AveragePricePerPixel  float64   `bigquery:"average_price_per_pixel" json:"averagePricePerPixel"`
+	AveragePixelPerMinute float64   `bigquery:"average_pixel_per_minute" json:"averagePixelPerMinute"`
+}
+
 type BigQuery interface {
 	QueryUsageSummary(ctx context.Context, userID string, creatorID string, spec QuerySpec) (*UsageSummaryRow, error)
 	QueryUsageSummaryWithTimestep(ctx context.Context, userID string, creatorID string, spec QuerySpec) (*[]UsageSummaryRow, error)
+	QueryTotalUsageSummary(ctx context.Context, spec QuerySpec) (*[]TotalUsageSummaryRow, error)
```

Comment:
Mind creating a separate `QuerySpec` object for this query, or maybe not even include any "query object"? From what I understand it uses a very small sub-set of fields from the full-blown `QuerySpec`, could just be inlined on the function args (otherwise, a `QueryTotalSpec` should be fine as well)

It would seem like a big source of confusion to me for a function to receive a bunch of data as input but then only really use a small sub-set of it. Can be misleading when using it and also when making changes to it ("what am i missing here? where are all this stuff used?")